daisy_workflows/image_import/inspection/inspect_disk.py

```diff
@@ -14,6 +14,22 @@
 # limitations under the License.
 
 # Example of communicating a key-value pair back to Daisy:
-#   print("Status: <serial-output key:'partition_scheme' value:'mbr'>")
+#
+
+import os
```

Comment:
Here's the file you want to update. Since I'm still working on making it easier to run tests, I'm fine if you put this code directly in `inspect-disk.py`, and I'll backport it into the library later. Or you can put it in the library now. It's up to you!

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/daisy_workflows/image_import/inspection/inspect-disk.py

Reply:
Also, the associated README has directions for running the inspection workflow directly.